proto/greenfield/virtualgroup/events.proto

```diff
@@ -69,6 +69,18 @@ message EventSwapOut {
   uint32 successor_sp_id = 4;
 }
 
+message EventCompleteSwapOut {
+  uint32 storage_provider_id = 1;
+  uint32 global_virtual_group_family_id = 2;
+  repeated uint32 global_virtual_group_ids = 3;
+}
+
+message EventCancelSwapOut {
+  uint32 storage_provider_id = 1;
```

Comment:
If we want to associate to `EventSwapOut`, we need to compare all these fields. Wondering if there exists swap out `id` for this.